### PR TITLE
chore: added UIID to prevent concurrency error

### LINKED ISSUE #4091: iot/api-client/manager tests fail - Connection lost, ALREADY_EXISTS: Resource already exists in project.
Found in #4087 and many others.

```
------------------------------------------------------------
- testing iot/api-client/manager
------------------------------------------------------------
[ERROR] Tests run: 21, Failures: 0, Errors: 17, Skipped: 4, Time elapsed: 451.364 s <<< FAILURE! - in com.example.cloud.iot.examples.ManagerIT
[ERROR] com.example.cloud.iot.examples.ManagerIT.testMqttDeviceState  Time elapsed: 445.836 s  <<< ERROR!
org.eclipse.paho.client.mqttv3.MqttException: Connection lost
Caused by: javax.net.ssl.SSLException: Connection reset
Caused by: java.net.SocketException: Connection reset

[ERROR] com.example.cloud.iot.examples.ManagerIT.testHttpDeviceConfig  Time elapsed: 0.367 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testMqttDeviceConfig  Time elapsed: 0.231 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testMqttDeviceEvents  Time elapsed: 0.338 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateGetDevice  Time elapsed: 0.481 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testPatchEs  Time elapsed: 0.389 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateConfigureDevice  Time elapsed: 0.317 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateGetRegistry  Time elapsed: 0.346 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateListDevices  Time elapsed: 0.204 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateDeleteEsDevice  Time elapsed: 0.304 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testHttpDeviceEvent  Time elapsed: 0.316 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testHttpDeviceState  Time elapsed: 0.296 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testGetIam  Time elapsed: 0.47 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateDeleteRsaDevice  Time elapsed: 0.202 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testCreateDeleteUnauthDevice  Time elapsed: 0.294 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testSetIam  Time elapsed: 0.301 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] com.example.cloud.iot.examples.ManagerIT.testPatchRsa  Time elapsed: 0.403 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=java-pst-16035045924).

[ERROR] Errors:
[ERROR]   ManagerIT.testCreateConfigureDevice » AlreadyExists io.grpc.StatusRuntimeExcep...
[ERROR]   ManagerIT.testCreateDeleteEsDevice » AlreadyExists io.grpc.StatusRuntimeExcept...
[ERROR]   ManagerIT.testCreateDeleteRsaDevice » AlreadyExists io.grpc.StatusRuntimeExcep...
[ERROR]   ManagerIT.testCreateDeleteUnauthDevice » AlreadyExists io.grpc.StatusRuntimeEx...
[ERROR]   ManagerIT.testCreateGetDevice » AlreadyExists io.grpc.StatusRuntimeException: ...
[ERROR]   ManagerIT.testCreateGetRegistry » AlreadyExists io.grpc.StatusRuntimeException...
[ERROR]   ManagerIT.testCreateListDevices » AlreadyExists io.grpc.StatusRuntimeException...
[ERROR]   ManagerIT.testGetIam » AlreadyExists io.grpc.StatusRuntimeException: ALREADY_E...
[ERROR]   ManagerIT.testHttpDeviceConfig » AlreadyExists io.grpc.StatusRuntimeException:...
[ERROR]   ManagerIT.testHttpDeviceEvent » AlreadyExists io.grpc.StatusRuntimeException: ...
[ERROR]   ManagerIT.testHttpDeviceState » AlreadyExists io.grpc.StatusRuntimeException: ...
[ERROR]   ManagerIT.testMqttDeviceConfig » AlreadyExists io.grpc.StatusRuntimeException:...
[ERROR]   ManagerIT.testMqttDeviceEvents » AlreadyExists io.grpc.StatusRuntimeException:...
[ERROR]   ManagerIT.testMqttDeviceState » Mqtt Connection lost
[ERROR]   ManagerIT.testPatchEs » AlreadyExists io.grpc.StatusRuntimeException: ALREADY_...
[ERROR]   ManagerIT.testPatchRsa » AlreadyExists io.grpc.StatusRuntimeException: ALREADY...
[ERROR]   ManagerIT.testSetIam » AlreadyExists io.grpc.StatusRuntimeException: ALREADY_E...
[ERROR] Tests run: 21, Failures: 0, Errors: 17, Skipped: 4
```

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes #4091 

- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java [hunk 1]
```diff
@@ -52,10 +53,16 @@
   private static final String CLOUD_REGION = "us-central1";
   private static final String ES_PATH = "resources/ec_public.pem";
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String REGISTRY_ID = "java-reg-" + (System.currentTimeMillis() / 100L);
+  private static final String REGISTRY_ID =
+      "java-reg-"
+          + (System.currentTimeMillis() / 100L);
```
Comment (lesv) [inline iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java:58]: If two or more of these started at the same time, there could be duplicates.  Would that be ok?  If not, you should also add a UUID here.

File: iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java [hunk 2]
```diff
@@ -52,10 +53,16 @@
...
   private static final String RSA_PATH = "resources/rsa_cert.pem";
   private static final String PKCS_PATH = "resources/rsa_private_pkcs8";
-  private static final String TOPIC_ID = "java-pst-" + (System.currentTimeMillis() / 100L);
+  private static final String TOPIC_ID =
+      "java-pst-"
+          + (System.currentTimeMillis() / 100L)
+          + "-"
+          + UUID.randomUUID().toString().substring(0, 10);
```
Comment (lesv) [inline iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java:65]: Seems like overkill to have currentTimeMillis + uuid, why not just UUID()?

**Review by lesv — APPROVED** [review]
Consider the questions

File: iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java (hunk 1 above)
Comment (munkhuushmgl) [reply iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java:58]: there is clearRegistries function that deletes the registries based on created Date (from oldest to new ones).

File: iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java (hunk 2 above)
Comment (munkhuushmgl) [reply iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java:65]: Hmm, I do not know why it was done this way, but I will just replace with just UUID()

File: iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java (hunk 1 above)
Comment (lesv) [reply iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java:58]: You might be better off using `java-reg-UUID-TimeMillis`.

**Comment by lesv** [comment]
@eaball35 Take a look at this.

**Comment by lesv** [comment]
@munkhuushmgl Mike - it's not passing J11 & J8

**Comment by munkhuushmgl** [comment]
> @munkhuushmgl Mike - it's not passing J11 & J8

Fixing it right now

**Review by lesv — APPROVED** [review]

**Pull request merged**